### PR TITLE
Rails Capstone Project: Submission

### PR DESCRIPTION
**Pull request by preciousuwhubetine**
This PR is for the submission of the rails capstone project.

Here's a brief overview of the features I implemented:
1. Data modeling: I created models for the different entities in the application.
2. User Authentication: Users get authenticated before accessing pages in the application.
3. Route Authorization: Users can access only routes they have sufficient permission to.
4. Testing: I wrote models, requests, and view specs.
5. Styling: The UI was styled using SASS.

Here's my presentation of the project: [Project Presentation Video](https://www.loom.com/share/b20adeef28a64d8386961c5ec9887b5a?sid=24de9033-8d44-4ad2-8f7b-32e6df82ec0c).

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
-    <title>MicroverseBudgetApp</title>
+    <title>Wallet Wise</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "global", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "splash", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "auth", "data-turbo-track": "reload" %>
```
Comment (VICTOR-CHUKWU) [inline app/views/layouts/application.html.erb:11]: - [ ] nice styles :+1:

File: app/views/categories/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <% end %>
+  <a>Create a New Category</a>
+</nav>
+
+<main>
+  <%= form_for(@category, url: categories_path(@category)) do |f| %>
+    <div class="picture-upload">
+      <label for="icon-upload">
+        <%= image_tag 'file.png', id:"category-icon" %>
+      </label>
```
Comment (VICTOR-CHUKWU) [inline app/views/categories/new.html.erb:13]: - [ ] good work implementing image upload :clap:

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (VICTOR-CHUKWU) [inline app/models/user.rb:3]: - [ ] it would be nice if user has to confirm his email after signup before proceeding to login :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @preciousbetine ,
# STATUS: APPROVED! 🥇 💯 
- [ ] This is actually one of the best project I have reviewed. Bravo!!! 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by preciousuwhubetine** [comment]
@VICTOR-CHUKWU Thanks for the review! I will consider implementing email confirmation in a future update.

**Pull request merged**